training-react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -0,0 +1,37 @@
+import { isArray } from './utils';
+
+export function min(param, ...arg) {
+  let val;
+  if (param !== undefined && param !== null) {
+    if (isArray(param)) {
+      val = Math.min(...param);
+    } else if (arg.length > 0) {
+      arg.push(param);
+      val = Math.min(...arg);
+    } else {
+      val = param;
+    }
+  }
+  return val;
+}
+
+export function copy(data) {
+  let copyArr;
+  if (isArray(data)) {
+    copyArr = [...data];
+  } else {
+    copyArr = { ...data };
+  }
+  return copyArr;
+}
+
+export function reverseMerge(arr1, arr2) {
```

Comment:
You can replace the `push` function for `concat` function, see this [documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat)